### ISSUE TITLE
Sometimes forms are not sent automatically 

### ISSUE THREAD

**Issue by dbemke** [getodk/collect#5473]
#### ODK Collect version
the store version 2022.4.4, the master version e8d2ebddc9f7f3a9c43822a3a1a32ea9b94ab304
#### Android version
10, 11, 13
#### Device used
Redmi 9T, Galaxy M12, Galaxy A32
#### Problem description
I’ve noticed that sometimes forms are not sent automatically. I don’t know which factor causes this issue but I was able to create steps to reproduce the issue.
After following the steps below, the second form is not sent and it is possible to edit the form and in "Send Finalized Form” it doesn’t have the red icon as if it is not being sent. 
#### Steps to reproduce the problem
1. Go to NewAppearance form.
[NewAppearance.xml.txt](https://github.com/getodk/collect/files/10899397/NewAppearance.xml.txt)
2. In "Image” questions tap "choose image” and add a 27MB picture/file.
3. Finalize and send the form.
4. Repeat steps 1-3 quickly (finalize and send a form before the first form is sent).
5. Check if the forms were sent automatically.

#### Expected behavior
All forms should be sent automatically.
#### Other information 
If the user goes to "Send Finalized Form” and manually sends the second form, the form is sent but there is no information about "background processes running” (the form is not being sent in the background).
If the user (after steps 1-5) finalizes and sends a third form, all forms will be sent automatically.

Please let me know if you need the file to add.

**Comment by seadowg** [comment]
@dbemke could you attach the file here as well please?

**Comment by dbemke** [comment]
The file is too big to attach here so I added access to a folder where you can find it.
https://drive.google.com/drive/folders/1IeuxQZU8Sw90PAOnTRlMToB3Wp_LuCoN?usp=sharing

**Comment by grzesiek2010** [comment]
This happens because we decided not to start a new job (for sending forms) if there is already one running. So basically what happens is:
1. A user finalizes a form with a big attachment.
2. A job for auto-sending is started (it takes some time because of the size of the form)
3. The user finalizes another form and a new job is not started because there is already one not finalized.
4. The first job is finished so the first form is uploaded.
5. The second form stays unsent because it wasn't taken into account when the first job was started (it didn't exist yet) and the second job was not started at all.
6. To send such a form we have to do that manually or finalize another form.

This works like that because as I said we implemented it using [KEEP ExistingWorkPolicy](https://developer.android.com/reference/androidx/work/ExistingWorkPolicy). There is also [APPEND](https://developer.android.com/reference/androidx/work/ExistingWorkPolicy#APPEND) that would be helpful in this case so we might want to consider it.
However, I think this is not important for now. It has worked like that for a long time so we can add it to v2023.2 but there is no need to fix it as a part of v2023.1. @lognaturel @seadowg please let me know if you agree.

**Comment by lognaturel** [comment]
Good find! I agree it’s likely an edge case but it’s more likely to happen on a poor connection. Using append sounds like an improvement and yes, it could come in .2.

**Comment by seadowg** [comment]
Yup sounds good to me. Will move this to v2023.2. The docs for the different `ExtistingWorkPolicy` values is a little hard to follow, but I'm thinking we might need [`APPEND_OR_REPLACE`](https://developer.android.com/reference/androidx/work/ExistingWorkPolicy#APPEND_OR_REPLACE) rather than just `APPEND` because of the different failure behaviour, but it's something we can experiment with when implementing.
